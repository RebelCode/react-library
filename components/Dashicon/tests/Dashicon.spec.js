import React from "react";
import { render, cleanup } from "@testing-library/react";
import { Dashicon } from "../index";

afterEach(cleanup);

describe("Dashicon", () => {
  it("should take a snapshot", () => {
    const { asFragment } = render(<Dashicon icon="menu" />);

    expect(asFragment(<Dashicon icon="menu" />)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(<Dashicon icon="menu" />);

    expect(queryByLabelText("Dashicon")).toBeTruthy();
  });
});
