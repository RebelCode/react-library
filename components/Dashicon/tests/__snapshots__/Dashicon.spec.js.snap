// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashicon should take a snapshot 1`] = `
<DocumentFragment>
  <span
    aria-label="Dashicon"
    class="dashicons dashicons-menu"
  />
</DocumentFragment>
`;
